applications/CompressiblePotentialFlowApplication/tests/test_CompressiblePotentialFlowApplication.py

```diff
@@ -31,8 +31,6 @@ def AssembleTestSuites():
     # smallSuite will contain the following tests:
     # - testSmallExample
     smallSuite = suites['small']
-    smallSuite.addTest(PotentialFlowTests('test_SmallLiftJumpTest'))
-    smallSuite.addTest(PotentialFlowTests('test_LiftAndMoment'))
     smallSuite.addTest(PotentialFlowTests('test_Naca0012SmallAdjoint'))
```

Comment:
Are you checking everything in one test now? 

We could then rename it to "test_Naca0012Small", right?

Reply:
Yes, so I thought that since for the adjoing we first need to solve the primal, why not checking the results directly there.

I find the renaming a good idea. Actually I wanted to ask you. Does it makes sense that we call it test_RhombusSmall since the geometry is actually not a Naca?

Reply:
Its fine for me!